plugins/org.yakindu.sct.doc.user/src/Part2-User-Guide/40_editing_statecharts.textile

```diff
@@ -61,8 +61,13 @@ The canvas is the statechart editor's drawing area. When you create a new statec
 
 The following list gives an overview of what kind of actions you can perform on the canvas:
 * Add or remove a region
+<<<<<<< HEAD
+** To add a region to the canvas, select _Region_ in the "editor palette":#editor-palette, then click on the canvas location you want to place the region.
+** To remove a region from the canvas, select the region, then press the @[Del]@ key or select _Edit → Delete_ from the menu bar.
+=======
 ** To add a region to the canvas, select _Region_ in the "editor palette":#editor-palette, then click at the canvas location you want to place the region.
 ** To remove a region from the canvas, select the region, then press @[Del]@ or select _Edit → Delete_ from the menu bar.
+>>>>>>> master
```

Comment:
This indicates a merge conflict on your side. Please resolve this conflict first!